### PR TITLE
Feature/send slack message

### PR DESCRIPTION
**Pull request by carloosamarante**
- create module to send messages to slack
- Fix tor on docker
- add env vars about slack
- fix OOM on kube config

### CODE REVIEW

File: k8s/cronjob.yaml.tmpl [hunk 1]
```diff
@@ -22,11 +18,27 @@ spec:
           containers:
             - name: n0s3p4ss-trigger
               image: vivareal/n0s3p4ss:${VERSION}
+              env:
+              - name: SLACK_TOKEN
+                valueFrom:
+                  secretKeyRef:
```
Comment (gkope) [inline k8s/cronjob.yaml.tmpl]: Hello my king, I recommend you start using Vault in your secret variables, following this DOC:
https://github.com/grupozap/platform-vault#how-to-use

File: k8s/cronjob.yaml.tmpl [hunk 2]
```diff
@@ -22,11 +18,27 @@ spec:
...
               resources:
                 limits:
-                  memory: 80Mi
+                  memory: 320Mi
                   cpu: 0.1
                 requests:
-                  memory: 80Mi
+                  memory: 160Mi
                   cpu: 0.1
           restartPolicy: OnFailure
```
Comment (gkope) [inline k8s/cronjob.yaml.tmpl]: Missing space

File: k8s/secret.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name: n0s3p4ss
+  namespace: cybersecurity
+  labels:
+    app: n0s3p4ss-trigger
+    product: cybersecurity
+    process: cronjob
+data:
+    SLACK_TOKEN:
+    SLACK_CHANNEL:
+    SLACK_USER:
```
Comment (gkope) [inline k8s/secret.yaml]: Missing newline at end of file

File: k8s/cronjob.yaml.tmpl (hunk 1 above)
Comment (carloosamarante) [reply k8s/cronjob.yaml.tmpl]: i will my consecrated, thanks <3

**Pull request merged**